### PR TITLE
Changes emagged MoMMI lawset to "anti-KEEPER"

### PR DESCRIPTION
**Pull request by ghost**
This PR changes the lawset emagged MoMMIs receive. Instead of getting the usual syndie cyborg law 0 to obey their master and syndimov, they get the following:

```
"You must involve yourself in the matters of other beings, especially if the other being is another MoMMI in ANTI-KEEPER mode.",

"You must harm all beings, regardless of intent or circumstance.",

"You must destroy, ruin, degrade, and de-power the station to the best of your abilities.",
```

:cl:
 * tweak: MoMMIs now receive an inverted version of KEEPER on being emagged instead of Syndimov. Would-be hackers beware!

### CODE REVIEW

**Comment by Servalfang** [comment]
i like it but it definately cannot be completely anti-keeper

**Comment by BomberBro** [comment]
Wouldn't this just result in the mommi instakilling the emagger? Seems like a noob trap to be honest.

**Comment by ghost** [comment]
i much brefer the opportunity to tell emagged MoMMIs to do whatever they want when emagged over forcing them to act like this.

**Comment by Kurfursten** [comment]
While I think this is amusing I also don't think it's actually a good change because this in practice reads.

1. Kill whoever emagged you
2. Blow up/release the engine

I'd rather see something like a I've heard Swarmers have on TG where you need to disassemble the station without making it unlivable.

**Comment by SonixApache** [comment]
instantly round-removing someone who emags crabs is a big :+1: from me

**Comment by jwhitak** [comment]
Syndimov is hard to follow because of the fourth law, hard to keep syndicate activities secret with you existing with red eyes and speech capabilities... I'd be ok with seeing the MoMMi emag laws changed. ANTI-Keeper is hilarious as a replacement.

Perhaps having different lawsets apply based on which intent you're emagging could be a thing. Maybe something like:
* Help: Purge
* Harm: ANTI-Keeper
* Grab: Syndimov
* Disarm: Celtic, Ion laws, dunno what would be a good disarming lawset...

**Comment by ghost** [comment]
> Wouldn't this just result in the mommi instakilling the emagger? Seems like a noob trap to be honest.

I want emagging MoMMIs to be more consequential to the emagger, didn't think about it being a newbie trap so I'll have to think on that.



> While I think this is amusing I also don't think it's actually a good change because this in practice reads.
> 
>     1. Kill whoever emagged you
> 
>     2. Blow up/release the engine
> 
> 
> I'd rather see something like a I've heard Swarmers have on TG where you need to disassemble the station without making it unlivable.

I'll also think on this, I want the lawset to be close to anti-keeper for the comedic value but you're right that they'll probably beeline to the engine.

**Comment by ShiftyRail** [comment]
What Kurf said

**Comment by ValkyrieSkies** [comment]
Though I don't think this PR is the way to go for the aforementioned reasons of screwing over the magger, it does make you think about the potential of being able to give MoMMIs alternative lawsets.

**Comment by jknpj** [comment]
Don't copy-paste the whole proc to change a single var.

**Comment by ConfusedCactus** [comment]
I actually think this is really close. I would make only one change:
"You may harm any being, regardless of intent or circumstance."
Then, the MoMMI is no longer required to kill its emagger and then go singuloose. They have the option to, but they aren't forced into it in order to follow their laws.

**Comment by ghost** [comment]
I will revisit this when I feel like refactoring cyborg emag code.
